src/lib/trust_system/peer_trust.ml

```diff
@@ -138,18 +140,26 @@ module Make0 (Inputs : Input_intf) = struct
     let%map () =
       match (simple_old.banned, simple_new.banned) with
       | Unbanned, Banned_until expiration ->
+          let message =
+            if tmp_bans_are_disabled then
+              "Would ban peer $peer_id until $expiration due to $action, \
+               refusing due to trust system being disabled"
```

Comment:
isn't the banning that's disabled, not the entire trust system, which calculates trust scores, etc.